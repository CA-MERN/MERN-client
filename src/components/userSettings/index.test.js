import { shallow } from 'enzyme';
import React from 'react';

import UserSettings from '.';

it('renders', () => {
    expect(shallow(<UserSettings />)).toMatchSnapshot();
});

it('expects text to render', () => {
    expect(shallow(<UserSettings />).text()).toMatch('First NameLast Name')
})
