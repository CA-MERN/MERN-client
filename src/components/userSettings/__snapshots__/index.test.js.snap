// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <h1>
    User Settings:
  </h1>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="photo"
    >
      Photo
    </label>
    <input
      id="file"
      name="file"
      onChange={[Function]}
      onSubmit={[Function]}
      type="file"
    />
    <label
      htmlFor="name"
    >
      Name
    </label>
    <input
      id="userSettingsName"
      name="name"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    <label
      htmlFor="email"
    >
      Email Address
    </label>
    <input
      id="userSettingsEmail"
      name="email"
      onBlur={[Function]}
      onChange={[Function]}
      type="email"
      value=""
    />
    <label
      htmlFor="password"
    >
      Password
    </label>
    <input
      id="userSettingsPassword"
      name="password"
      onBlur={[Function]}
      onChange={[Function]}
      type="password"
      value=""
    />
    <label
      htmlFor="confirmPassword"
    >
      Password
    </label>
    <input
      id="userSettingsConfirmPassword"
      name="confirmPassword"
      onBlur={[Function]}
      onChange={[Function]}
      type="password"
      value=""
    />
    <button
      type="submit"
    >
      Update Details
    </button>
  </form>
</div>
`;
